packages/editor/src/store/effects/reusable-blocks.js

```diff
@@ -131,13 +133,14 @@ export const saveReusableBlocks = async ( action, store ) => {
 			id,
 		} );
 		const message = isTemporary ? __( 'Block created.' ) : __( 'Block updated.' );
-		dispatch( createSuccessNotice( message, { id: REUSABLE_BLOCK_NOTICE_ID } ) );
+		dataDispatch( 'core/notices' ).createSuccessNotice( message, {
```

Comment:
It makes me wonder if we should refactor effects to take `data` as param rather than `store`. This would align closer to what we have in other parts of the applications, in particular, `withSelect` and `withDispatch`.

Reply:
> It makes me wonder if we should refactor effects to take `data` as param rather than `store`. 

By date, would you mean the current registry? I guess that would work as a solution for https://github.com/WordPress/gutenberg/pull/9617/files#r228623495, but as noted in that comment, we'll need a similar solution for selectors and controls as well.

Reply:
Yes, this is what I thought.